### PR TITLE
Add ARM64 support for Travis-CI

### PR DESCRIPTION
**Pull request by odidev**
Added ARM64 job in Travis-CI for openjdk8, openjdk11 and graalvm openjdk11.

Signed-off-by: odidev <odidev@puresoftware.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,10 +25,24 @@ jobs:
       name: "OpenJDK 8"
       jdk: openjdk8
       script: mvn clean verify -B
+    - stage: test
```
Comment (slinkydeveloper) [inline .travis.yml:28]: Can we enable this only on master?

Comment (slinkydeveloper) [reply .travis.yml:28]: Why stage:test here too?

File: .travis.yml [hunk 2]
```diff
@@ -22,10 +25,24 @@ jobs:
...
+      addons:
+        apt:
+          - openjdk-8-jdk
+      jdk: openjdk8
+      script: mvn clean verify -B
     - name: "OpenJDK 11"
       jdk: openjdk11
       script: mvn clean verify -B
-    # GraalVM
+    - name: "OpenJDK 11"
```
Comment (slinkydeveloper) [inline .travis.yml:40]: Can we enable this only on master?

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 language: java
 
 before_install:
+  - if [[ "${TRAVIS_CPU_ARCH}" == "arm64" ]]; then
```
Comment (slinkydeveloper) [inline .travis.yml:4]: Why do we need to install maven from apt-get?

**Review by slinkydeveloper — COMMENTED** [review]
May I ask why we need this? All the api/core modules doesn't depend on any transitive jni code

**Comment by bsideup** [comment]
Just curious... What's the point of testing a pure Java library on ARM64 when JVM's ARM64 compatibility is already tested? Won't it add an unnecessary load to the CI for... nothing?

File: .travis.yml (hunk 3 above)
Comment (bsideup) [reply .travis.yml:4]: Especially given that there is a Maven Wrapper in the project that should be unused instead

**Comment by odidev** [comment]
Although this is a pure java library but while i was testing it in travis i found out that travis is using graal vm for testing, these graalvm binaries mentioned can only be tested for amd64 platforms. I have updated the graalvm binaries link for the testing for arm64 platform.

File: .travis.yml (hunk 1 above)
Comment (odidev) [reply .travis.yml]: I have enable the openjdk tests only for master branch and removed any reductant tags (such as stage)

File: .travis.yml (hunk 2 above)
Comment (odidev) [reply .travis.yml]: I have enabled the openjdk arm64 tests for master only

File: .travis.yml (hunk 3 above)
Comment (odidev) [reply .travis.yml]: I have used the maven wrapper present in this.

**Comment by slinkydeveloper** [comment]
@odidev the graalvm tests we do run in vm mode, they don't compile and test our application in native mode

**Comment by slinkydeveloper** [comment]
This is not valid anymore because we migrated to gh actions

**Pull request closed without merging**